src/Features/Core/Portable/UseNullPropagation/AbstractUseNullPropagationDiagnosticAnalyzer.cs

```diff
@@ -49,18 +50,27 @@ protected AbstractUseNullPropagationDiagnosticAnalyzer()
         protected abstract ISyntaxFactsService GetSyntaxFactsService();
         protected abstract ISemanticFactsService GetSemanticFactsService();
 
+        protected abstract bool TryAnalyzePatternCondition(SyntaxNode conditionNode, out SyntaxNode conditionLeft, out SyntaxNode conditionRight, out bool isEquals);
+
         protected override void InitializeWorker(AnalysisContext context)
         {
             context.RegisterCompilationStartAction(startContext =>
             {
                 var expressionTypeOpt = startContext.Compilation.GetTypeByMetadataName("System.Linq.Expressions.Expression`1");
+
+                var objectType = startContext.Compilation.GetSpecialType(SpecialType.System_Object);
+                var referenceEqualsMethod = objectType?.GetMembers(nameof(ReferenceEquals))
+                                      .OfType<IMethodSymbol>()
```

Comment:
try to align these with the previous line.

Reply:
➡️ No need to align expressions that the editor doesn't align automatically (which is only the LINQ syntax). If you adjust the indentation, just make it a single indentation, like this:

```csharp
var referenceEqualsMethod = objectType?.GetMembers(nameof(ReferenceEquals))
    .OfType<IMethodSymbol>()
```

Reply:
Note: we do often try to align long chains of linq-as-method calls.  In C# we try to align on the ```.```, and in VB we put the dot on the previous line and align on the name.  It would be nice to preserve that.

That said, going with single indentation is also acceptable.  It's usually strange indentation like this that we try to avoid.

Reply:
I originally had the dots lined up but I think I renamed the variable at some point. I put things back so the dots are lined up.